nodeEditor/src/diagram/graphFrame.ts

```diff
@@ -1415,9 +1426,35 @@ export class GraphFrame {
         } else {
             newFrame.refresh();
         }
+        
+        
+        for (var node of newFrame.nodes) {
+            for (var port of node.outputPorts) { // Output
+                if(port.exposedOnFrame) {
+                    port.isExposed = true;

```

Comment:
why do we have isExposed and exposedonFrame?

Reply:
The difference is one is weather it actually is exposed or is the user desires it to be exposed. In theory it would make sense to change the variable name now but the json export name can't change cause of backwards compatibility.

Reply:
ok makes sense! thanks